server/src/test/java/io/crate/execution/engine/collect/ShardCollectorProviderTest.java

```diff
@@ -54,25 +54,6 @@ public void assertNoShardEntriesLeftInShardCollectSource() throws Exception {
         }
     }
 
-    @Test
-    public void testClosedIndicesHaveNoShardEntries() throws Exception {
```

Comment:
The shards associated with a closed table will still be part of of the collect source.

Reply:
Shouldn't we replace this with the opposite test case, ensuring that shard entries exists even for closed tables?